_posts/2021-03-25-mark-turner.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "From Engineer to Manager"
+author: "@pweschmidt"
+rse: "Mark Turner"
+excerpt: "Meet Mark Turner, Head of the RSE team at Newcastle University in the UK. In this episode we discuss the transition from engineer to manager and how to build effective RSE teams."
+date: 2021-03-25 8:30:00
+external_media: https://us-rse.org/rse-stories-episodes-1/2021/rse-stories-Mark-Turner-episode-55.mp3
```

Comment:
Looks like the episode number is correct here, phew! Sorry about that - I must have not looked at our episode calendar for Ron's episode.

Reply:
Oh you just renamed the episode - awesome thank you! Super speedy :)